model/user.py

```diff
@@ -17,6 +17,7 @@ def __init__(self, slack_id):
         self.biography = ""
         self.image_url = ""
         self.permissions_level = Permissions.member
+        self.karma = 1
```

Comment:
Perhaps starting with 0 karma would make more sense?

Reply:
I was thinking like reddit where everyone starts off with 1 karma, but I can roll with 0 if that's what we want.

Reply:
Fair enough, that makes sense.